localboot/main.go

```diff
@@ -123,7 +124,20 @@ func BootGrubMode(devices []storage.BlockDev, baseMountpoint string, guid string
 	if len(bootconfigs) == 0 {
```

Comment:
since you're changing this file, can you please also move this `if` above? Just before the log.Printf at line 120. It doesn't make much sense here

Reply:
done